patches/llvm-project-perf/0000-LTOpasses-add-loop-unroll.patch

```diff
@@ -0,0 +1,55 @@
+From 4adfc5231d2c0182d6278b4aa75eec57648e5dd4 Mon Sep 17 00:00:00 2001
+From: Vladi Krapp <vladi.krapp@arm.com>
+Date: Tue, 3 Sep 2024 14:12:48 +0100
+Subject: [Pipelines] Additional unrolling  in LTO 
+
+Some workloads require specific sequences of events to happen
+to fully simplify. This adds an extra full unrolling pass to help these
+cases on the cores with branch predictors. It helps produce simplified
+loops, which can then be SROA'd allowing further simplification, which
+can be important for performance.
+This is added under own flag - spending extra compile time to get extra
```

Comment:
I would name the flag here in the description, so people don't have to hunt through the commit itself to find it.

under own flag -> under it's own flag

these comments also apply to the description of this patch itself, which has the same text